### ISSUE TITLE
Questions about Optimizers, Loss and Light Directions

### ISSUE THREAD

**Issue by sungam94** [Univr-RTI/NeuralRTI#2]
Hey, I tried to implement it myself in pytorch, and I'm not the most fluent in keras, so i got a few questions about the implementation.
Do you use one or two optimizer for training?
Is the loss a mean of multiple light directions?
In other words, does one batch mix light directions during training?
